flaml/automl/automl.py

```diff
@@ -2,6 +2,7 @@
 #  * Copyright (c) FLAML authors. All rights reserved.
 #  * Licensed under the MIT License. See LICENSE file in the
 #  * project root for license information.
+from __future__ import annotations
```

Comment:
Please remove the changes for this file.

Reply:
done :).

Do you think there's a way to allow for type annotation across the codebase?
I think it'd help newcomers like me. Should I open an issue?

Reply:
Please feel free to open an issue to discuss that. I'm not clear what it means by "allow for type annotation". Is type annotation currently now allowed in the codebase?

Reply:
@coffepowered do you mean enabling modern annotation types without importing `annotations` from `__future__` within each module? Unfortunately, I don't know of any way of doing that. However, I do think importing `annotations` and updating the typing would make for a good PR. Probably best to do it module-by-module though, otherwise it would very difficult to review.